cub/device/dispatch/dispatch_radix_sort.cuh

```diff
@@ -51,6 +51,11 @@
 
 #include <thrust/system/cuda/detail/core/triple_chevron_launch.h>
 
+#if defined(__clang__)
```

Comment:
Can you add a comment here describing what error this is suppressing? We will likely want to remove this after cleaning up the `unroll` pragmas.

Reply:
done